hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/upgrade/OMPrepareRequest.java

```diff
@@ -245,12 +252,6 @@ public static void takeSnapshotAndPurgeLogs(long prepareIndex,
               "match specified purge index {}. ", actualPurgeIndex,
             snapshotIndex);
       }
-
```

Comment:
removing this check do have any impact?

Reply:
No.   Since `snapshotIndex >= prepareIndex` and `actualPurgeIndex == snapshotIndex`, we cannot have `actualPurgeIndex < prepareIndex`.